jobboerse/_posts/2023-02-03-infineon.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "Werkstudententätigkeit bei Infineon Technologies AG Warstein"
+author: Mira
+publish_until: 2023-03-02T00:00:00
+---
+Die Infineon Technologies AG Warstein, unweit von Dortmund, beschäftigt bereits viele aktuell Studierende und Absolvent:innen der TU Dortmund. 
+In der derzeit offenen [Stellenausschreibung](/dokumente/ausschreibungen_jobboerse/2023-02-03-infineon.pdf) findet ihr alle notwendigen Details zu einer Werkstudententätigkeit im Process Engineering Bereich.
```

Comment:
The path to the pdf is wrong. That's why the checks fail. Could you please fix this by either putting the pdf to the right place or changing it in the post.